distributed/scheduler.py

```diff
@@ -1357,7 +1355,7 @@ def update_graph(self, client=None, tasks=None, keys=None,
         if isinstance(user_priority, Number):
             user_priority = {k: user_priority for k in tasks}
 
-        priority = priority or order(tasks)  # TODO: define order wrt old graph
+        priority = priority or dask.order.order(tasks)  # TODO: define order wrt old graph
```

Comment:
Did you mean to double `.order` here?

Reply:
Yes, same as above.  This change is somewhat orthogonal to this PR, I just decided to start using namespaces more heavily rather than direct imports.